fluentd/configs.d/openshift/filter-k8s-meta.conf

```diff
@@ -3,7 +3,7 @@
   kubernetes_url "#{ENV['K8S_HOST_URL']}"
   cache_size "#{ENV['K8S_METADATA_CACHE_SIZE'] || '1000'}"
   watch "#{ENV['K8S_METADATA_WATCH'] || 'false'}"
-  use_journal "#{ENV['USE_JOURNAL'] || 'false'}"
+  use_journal "#{ENV['USE_JOURNAL'] || 'nil'}"
```

Comment:
auto-detect is a nice idea!  Could the default value 'nil' be a little more descriptive like 'auto-detect'?  Or should it be strictly 'nil'?

Reply:
It has to be `nil` - the fluentd config parser will turn the `'nil'` into a ruby `nil` value.

Reply:
I see. Thanks!